relay-pii/src/regexes.rs

```diff
@@ -106,186 +106,239 @@ macro_rules! ip {
+        ip!(v6s),
+        "|",
+        "(",
+        ip!(v6s),
+        ":){1,5}:(:",
+        ip!(v6s),
+        "){1,2}|",
+        "(",
+        ip!(v6s),
+        ":){1,4}:(:",
+        ip!(v6s),
+        "){1,3}|",
+        "(",
+        ip!(v6s),
+        ":){1,3}:(:",
+        ip!(v6s),
+        "){1,4}|",
+        "(",
+        ip!(v6s),
+        ":){1,2}:(:",
+        ip!(v6s),
+        "){1,5}|",
+        ip!(v6s),
+        ":((:",
+        ip!(v6s),
+        "){1,6})|",
+        ":((:",
+        ip!(v6s),
+        "){1,7}|:)|",
+        "fe80:(:",
+        ip!(v6s),
+        "){0,4}%[0-9a-zA-Z]{1,}",
+        "::(ffff(:0{1,4}){0,1}:){0,1}",
+        ip!(v4a),
+        "|",
+        "(",
+        ip!(v6s),
+        ":){1,4}:",
+        ip!(v4a),
+        ")([\\s]|[[:punct:]]|$)",
+    )
+);
 
 // http://www.richardsramblings.com/regex/credit-card-numbers/
 // Re-formatted with comments and dashes support
 //
 // Why so complicated? Because creditcard numbers are variable length and we do not want to
 // strip any number that just happens to have the same length.
-static CREDITCARD_REGEX: Lazy<Regex> = Lazy::new(|| {
-    Regex::new(
-        r#"(?x)
-        \b(
-            (?:  # vendor specific prefixes
-                  3[47]\d      # amex (no 13-digit version) (length: 15)
-                | 4\d{3}       # visa (16-digit version only)
-                | 5[1-5]\d\d   # mastercard
-                | 65\d\d       # discover network (subset)
-                | 6011         # discover network (subset)
-            )
+regex!(
+    CREDITCARD_REGEX,
+    r#"(?x)
+    \b(
+        (?:  # vendor specific prefixes
+                3[47]\d      # amex (no 13-digit version) (length: 15)
+            | 4\d{3}       # visa (16-digit version only)
+            | 5[1-5]\d\d   # mastercard
+            | 65\d\d       # discover network (subset)
+            | 6011         # discover network (subset)
+        )
 
-            # "wildcard" remainder (allowing dashes in every position because of variable length)
-            ([-\s]?\d){12}
-        )\b
-        "#,
-    )
-    .unwrap()
-});
+        # "wildcard" remainder (allowing dashes in every position because of variable length)
+        ([-\s]?\d){12}
+    )\b
+    "#
+);
 
-static PATH_REGEX: Lazy<Regex> = Lazy::new(|| {
-    Regex::new(
-        r"(?ix)
+regex!(
+    PATH_REGEX,
+    r"(?ix)
+        (?:
             (?:
-                (?:
-                    \b(?:[a-zA-Z]:[\\/])?
-                    (?:users|home|documents and settings|[^/\\]+[/\\]profiles)[\\/]
-                ) | (?:
-                    /(?:home|users)/
-                )
+                \b(?:[a-zA-Z]:[\\/])?
+                (?:users|home|documents and settings|[^/\\]+[/\\]profiles)[\\/]
+            ) | (?:
+                /(?:home|users)/
             )
-            (
-                [A–Za-z0–9'\.\ _!\#^~-]+
-            )
-        ",
-    )
-    .unwrap()
-});
+        )
+        (
+            [^/\\\r\n]+
```

Comment:
This is the change that fixes the regex with unicode disabled, while still meeting the requirements of https://github.com/getsentry/relay/pull/2724: https://github.com/getsentry/relay/blob/b5b738f4b2c9a546acaa67fb80026fe46bfadb41/relay-pii/src/builtin.rs#L1247-L1248